metsim/datetime.py

```diff
@@ -89,5 +90,7 @@ def units_from_freq(freq, origin=DEFAULT_ORIGIN):
         return 'hours since %s' % origin
     elif 'D' in freq:
         return 'days since %s' % origin
+    elif 'T' in freq:
+        return 'minutes since %s' % origin
     else:
-        raise NotImplementedError()
+        raise NotImplementedError('freq %s not supported at this time' % freq)
```

Comment:
Done, though the logging module doesn't require the `.format` method to be called. This works:

```python
logger.info('just a {} example', 'little')
```

xref: https://docs.python.org/3/howto/logging-cookbook.html#using-custom-message-objects